apps/settings/js/icc.js

```diff
@@ -61,6 +59,18 @@ require([
     var _backHandler = null;
     init();
 
+
+    function localizeElement(element, l10n) {
+      if (typeof(l10n) === 'string') {
+        element.setAttribute('data-l10n-id', l10n);
+      } else if (l10n.raw) {
+        element.removeAttribute('data-l10n-id');
+        element.textContent = l10n.raw;
+      } else {
+        navigator.mozL10n.setAttributes(element, l10n.id, l10n.attrs);
+      }
+    }
+
```

Comment:
We have this pattern documented here: https://developer.mozilla.org/en-US/Firefox_OS/Developing_Gaia/localization_code_best_practices#Writing_APIs_that_operate_on_L10nIDs

I'll add comments!